### DISCUSSION TITLE
Log how much data was sent to the client in loaders/actions

### DISCUSSION THREAD

**Discussion by alexdotpink** [Code-Forge-Net/Remix-Dev-Tools#97] [Ideas]
It would be incredibly useful to know how much data (in KB) the server is sending to the client on every request, both for loaders and for actions.

**Comment by AlemTuzlak** [comment]
I would love to do this and I'll move this to discussions because at the moment I'm not sure if I can read data without crashing the server. I'll add this in if I figure that part out!
